core-services/beneficiary-idgen/README.md

```
@@ -0,0 +1,44 @@
+### Service Dependencies
+
+- egov-mdms-service
+
+
+## Service Details
+
+The application can be run like any other Spring Boot application, but you must add the Lombok extension to your IDE to enable annotation processing.
+
+For IntelliJ IDEA, install the plugin directly. For Eclipse, add the Lombok JAR to the `eclipse.ini` file using:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)